src/twisted/application/runner/_pidfile.py

```diff
@@ -81,7 +86,7 @@ def __exit__(excType, excValue, traceback):
 @implementer(IPIDFile)
 class PIDFile:
     """
-    Concrete implementation of L{IPIDFile} based on C{IFilePath}.
+    Concrete implementation of L{IPIDFile} based on C{FilePath}.
```

Comment:
There is probably a good reason behind this, but it's not immediately obvious to me why there is a need here to use the concrete class rather than the interface.

Reply:
It's not "based on" so much as "using".  I'll remove the "based on" clause there, since that's implementation detail.